crawl-ref/source/dat/des/arrival/simple.des

```diff
@@ -2029,3 +2029,80 @@ x.G.G.G.x
 x.......x
 xxxx+xxxx
 ENDMAP
+
+###############################################################################
+
+NAME:   giann_horns   
+TAGS:   arrival no_monster_gen
+ORIENT: float
+MAP
+x@xx...................xx@x
+x..xx.................xx..x
+x...xx..G..xx+xx..G..xx...x
+x....xx...xx...xx...xx....x
+x.....xxxxx.....xxxxx.....x
+x............{............x
+xx.......................xx
+ xxxxxxxxxxxx+xxxxxxxxxxxx
+ENDMAP
+
+NAME:   giann_bridges
+TAGS:   arrival no_monster_gen
+MONS:   plant
+ORIENT: float
+MAP
+.........................
+.1wwwwwwww1...1wwwwwwww1.
+.wwwWWWWwww...wwwWWWWwww.
+.wwwWWWWwww...wwwWWWWwww.
+.1wwwwwwww.....wwwwwwww1.
+............{............
+.1wwwwwwww.....wwwwwwww1.
+.wwwWWWWwww...wwwWWWWwww.
+.wwwWWWWwww...wwwWWWWwww.
+.1wwwwwwww1...1wwwwwwww1.
+.........................
+ENDMAP
+
+NAME:   giann_guarded_fountains
+TAGS:   arrival no_monster_gen no_item_gen no_rotate
+MONS:   hobgoblin
+KITEM:  $ = gold q:1
+SUBST:  S = $.
+ORIENT: float
+MAP
+.......................
+.......................
+.......................
+.........1...1.........
+.......................
+xxxxxxxxxxx+xxxxxxxxxxx
+x........xG.Gx........x
+x...S....x...x....S...x
+x..STS...xx+xx...STS..x
+xx..S.............S..xx
+ xxxxxx....{....xxxxxx 
+      xx.......xx
+       xxxxxxxxx
+ENDMAP
+
+NAME:   giann_forest_graveyard
+TAGS:   arrival no_monster_gen
+MONS:   plant
+ORIENT: float
+: dgn.delayed_decay(_G, 'h', 'human skeleton')
```

Comment:
if you wanted to be extra clever, this could be a skeleton of the same species as the player -- the memento_mori arrivals vault for implementation details